hieradata/common.yaml

```diff
@@ -352,11 +352,18 @@ govuk::apps::content_audit_tool::db::backend_ip_range: "%{hiera('environment_ip_
 govuk::apps::content_audit_tool::redis_host: "%{hiera('sidekiq_host')}"
 govuk::apps::content_audit_tool::redis_port: "%{hiera('sidekiq_port')}"
 
+govuk::apps::content_performance_manager::rabbitmq::amqp_pass: 'content_performance_manager'
+govuk::apps::content_performance_manager::rabbitmq_password: 'content_performance_manager'
```

Comment:
Is this ok to be plaintext?

Reply:
We expect this to be overridden in other environments. However the CI tests only use this file and fail when something is not present in this file.

Reply:
It might be a bit neater to have the password default to undefined then.

Reply:
As discussed offline - I don't think we can make it undefined without changing how the tests work, so I'm going to leave this as is for now. Agree it's not ideal but I think we would need a larger refactoring to make this work better for all applications.